tutorial/docs/alarms-management.md

```diff
@@ -128,7 +142,7 @@ query{
 ```
   ![View_Alarm_States](./images/17-view-alarm-states.png)
 
-  There further things you can do with these queries such as filtering but that will be covered in the following tutorial.
+  There are additional things you can do with these queries such as filtering and these will be covered in the following sections.
```

Comment:
```suggestion
  There are additional things you can do with these queries such as filtering and these will be covered in the following tutorial.
```


this is referencing the next tutorial I plan to write. not sure whether just to remove this